src/empty-state/empty-state.ts

```diff
@@ -0,0 +1,19 @@
+import { html, LitElement } from "lit";
+
+export class EmptyState extends LitElement {
+  render() {
+    return html`
+      <div class="container">
+        <span class="leading">
+          <slot name="leading"></slot>
+        </span>
+        <span class="content">
+          <slot name="content"></slot>
+        </span>
+        <span class="trailing">
+          <slot name="trailing"></slot>
+        </span>
+      </div>
+    `
+  }
+};
```

Comment:
i think `icon` and `actions` are better namesfor `trailing` and `leading` slots. `leading` and `trailing` are too generic in this case.